scripts/eslint/v7.js

```diff
@@ -0,0 +1,217 @@
+module.exports = {
```

Comment:
for my own curiosity - how did you create this config?

Reply:
Started from here and then modified a bunch of things https://github.com/typescript-eslint/tslint-to-eslint-config

Reply:
You might also be interested in this page which has a pretty detailed mapping of tslint to eslint rules. https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md